### PR TITLE
Feature/e2e order confirm

### PR DESCRIPTION
**Pull request by esteban-gs**
Changes:
- new test cases for the order confirm flow
- the most important case to get us out the door: paypal button when fees
- added some `data-` attributes to the `full-order-view.component.html`. I was avoiding it at all costs, but I could't get cypress to find the cost elements

### CODE REVIEW

File: cypress/utils/helpers.ts [hunk 1]
```diff
@@ -17,8 +24,169 @@ export const getTodayPlus = (
...
+    skill: skill.text_EN,
+    hours: skill.minHour,
+    description: "",
+    requiresHeavyLifting: false,
+    hourlyWage: skill.wage,
+    transportCost: getExpectedCostForWorkerCount(2, aPaidTransportRule),
+    days: 1, // this is the default
+  };
+
+  return [wa, wa2];
```
Comment (esteban-gs) [inline cypress/utils/helpers.ts:191]: late-night madness (－‸ლ)

File: cypress.json [hunk 2]
```diff
@@ -9,5 +9,6 @@
   "env": {
     "macheteApiUrl": "http://localhost:4213/api",
     "macheteLegacyUrl": "http://localhost:4213"
-  }
+  },
+  "video": false
```
Comment (jcii) [inline cypress.json:13]: What happens if we video=true?

**Review by jcii — APPROVED** [review]

File: cypress/utils/helpers.ts (hunk 1 above)
Comment (jcii) [reply cypress/utils/helpers.ts:191]: you could  `wa2 = [...wa]` and save lines (I think; test it)

File: cypress.json (hunk 2 above)
Comment (esteban-gs) [reply cypress.json:13]: When true, it saves a video of the ui when the test runs. Turning it off makes it way faster

**Pull request merged**